### DISCUSSION TITLE
Audio dropping in stream

### DISCUSSION THREAD

**Discussion by savredigital** [AzuraCast/AzuraCast#5613] [General]
Hi,
after update streaming audio dropping.

please help

https://radiosavre.com:8000/radio.mp3

**Comment by Mozitux** [comment]
Hello, are you talking about loss of stream transmission for your listeners when you broadcast live?

If so, I had a similar problem. You must ensure that your sound cards / microphone transmit with the same sampling stream as your broadcasting / WebDj software.

For example, 44100 Hz for all audio devices and on your streaming software. Otherwise, there will be sound distortion and interruptions in your stream.
